sourcegraph/sourcegraph
# Wildcard [Charts]: Update charts package documentation

Closes https://github.com/sourcegraph/sourcegraph/issues/39878

## Background
It's been a while since the last time we updated documentation for our charts. This PR 
- Updates Readme.md file 
- Provides better charts examples 

It should help other devs to understand what our chart can do and what can't. 

## Test plan
- Check new readme for the chart package
- Make sure that CI is green 
- Check new storybook stories for the charts



## App preview:

- [Web](https://sg-web-vk-update-chart-documentation.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-cynjrjvolv.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
